vhdbuilder/packer/configure-windows-vhd.ps1

```diff
@@ -205,6 +205,20 @@ function Get-FilesToCacheOnVHD {
     }
 }
 
+function Get-ToolsToVHD {
+    # Rely on the completion of Get-FilesToCacheOnVHD
+    $cacheDir = "c:\akse-cache\tools"
```

Comment:
akse-cache cannot be used since it will be deleted in AKS Windows CSE script

Reply:
is it only used for temporary caching? I think that we should not do that for this purpose.

Reply:
If we use a temp folder in $env:temp, we do not need to clean up it since it will be cleaned up automatically.

Reply:
It seems like that you are using the current code logic to download the files in akse-cache and validate the signature. It is good for me.
My only suggestion is to expand the archive to the folder `c:\aks-tools\du`

Reply:
Updated, thank you